### ISSUE TITLE
Add supported AMI's for amazon

### ISSUE THREAD

**Issue by stevendborrelli** [CiscoCloud/microservices-infrastructure#866]
We should documents AMIs for Amazon available here: 

https://aws.amazon.com/marketplace/pp/B00O7WM7QW
```
US East (N. Virginia)      ami-96a818fe
US West (Oregon)          ami-c7d092f7
US West (N. California)      ami-6bcfc42e
EU (Frankfurt)              ami-7cc4f661
EU (Ireland)              ami-e4ff5c93
Asia Pacific (Singapore)  ami-aea582fc
Asia Pacific (Sydney)      ami-bd523087
Asia Pacific (Tokyo)      ami-89634988
South America (Sao Paulo) ami-bf9520a2
```

We could also include this in terraform sample and do a lookup in the module file for aws.tf and include this in the project

```
source_ami = "${lookup(var.region, var.ami)}"
```

**Comment by ryane** [comment]
there is some scenario where the AMIs we currently recommend (https://aws.amazon.com/marketplace/pp/B00O7WM7QW - release 2015/09/29) are not available in some AWS accounts. We need to test with the latest release of the official Centos 7 AMI

<img width="366" alt="screen-shot-2015-11-24-at-16 53 46" src="https://cloud.githubusercontent.com/assets/7346/11369013/6ff48d44-9282-11e5-942e-d87f1cd7ad3a.png">

**Comment by sehqlr** [comment]
Is the plan to support both the HVM and the official images?

**Comment by ryane** [comment]
The official centos 7 amis are HVM-based (see url above). All of the current AWS instance types support HVM. I don't think we need to support PV amis but if anyone knows of a reason, please chime in.

It does appear that the release we have been using (7.0.1406 - 2014-09-29, released 09/29/2014) is deprecated and not available for all AWS users. We need to test and support the newer release (7.1.1503 - 2015-09-28, released 10/13/2015). I just had a successful build with `ami-d440a6e7` in us-west-2.

**Comment by sehqlr** [comment]
I was able to build with eu-west-1 in #870.

**Comment by ryane** [comment]
merged in #870
